config/initializers/job_configurations.rb

```diff
@@ -1,3 +1,4 @@
+cron_1m = '0/1 * * * *'
```

Comment:
I'm not _super_ familiar with the usual patterns here, but a minute is pretty frequent, would somewhere in the range of 5-15m intervals be sufficient?